### ISSUE TITLE
Note: 03-08-25 Meeting Notes

### ISSUE THREAD

**Issue by g-simmons** [g-simmons/persona-research-internship#237]
Meeting notes

Logan

Blog post -- ready to go on website, needs links for references etc.

Multiple trials for GPU vs. non-GPU implementation of store_matrices

- Install the correct torch version (search github issues for pytorch)
- 5 trials is fine

Harry

Issues with 2.8B and 12B heatmaps -- dimension mismatch

Finished plots for other model sizes (70M, 160M, 1.4B)

Need to choose some terms, otherwise plot is unreadable

Observations
- Scores decrease vs. training steps
- Linear rep score has different y axis
- Some terms have drastically worse linear rep scores than other terms
- Lowest linear rep score is 0.5
- Smaller models seem to have rougher score trajectories (more oscillation)
- Linear rep score looks rougher, other scores look somewhat smoother
- Better representations with more parameters
- Scores change a lot for 70M vs. 1.4B. Patterns look similar but absolute values differ.
- In most cases, term rankings for representation quality are consistent across model sizes. There is an expection for communication and location for model sizes 70M and 160M.
- In most cases, terms that are better-represented early in training tend to be better-represented later in training. There are exceptions (communication and location for 70M).
- Maybe small models struggle with having enough representational capacity, this could explain why scores cross for 70M but less for other models.
- Scores change non-monotonically in a way that is not just due to noise. 160M representations get better then worse from 100k steps to 120k steps to 140k steps.

How does variation for individual term scores compare to variation for overall scores?

Next steps:
- Save the data for the plots
- Write up the observations, add the figure and observations to the blog post. (persona-research-internship/llm_persona_website/ontology_topology.html)
  - Add to Research Question 4: Do deeper ontology terms get improved representations later in training?

Raymond

Typical dimension for the multiword case is 368

Never did multiword for 12B

160 is the old dimension for non-multiword
